gluten-delta/src/main/scala/io/glutenproject/execution/DeltaScanTransformer.scala

```diff
@@ -14,9 +14,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.glutenproject.extension
+package io.glutenproject.execution
```

Comment:
So `extension` was by mistake?

Reply:
I think so. I synced with @YannByron and he will take a further look as there may still be some other issues related with column mapping.